web/src/hooks/useColors.ts

```diff
@@ -1,10 +1,20 @@
 import { useColorModeValue } from "@chakra-ui/react";
+import { generateKeyPair } from "crypto";
 
 export const useColors = () => {
+  const headerBg = useColorModeValue("gray.200", "gray.700");
+  const optionBg = useColorModeValue("gray.200", "gray.700");
+  const optionBgHover = useColorModeValue("gray.300", "gray.800");
+  const optionCorrect = useColorModeValue("green.400", "green.600");
+  const optionIncorrect = useColorModeValue("red.400", "red.600");
+    const primaryColorHover = useColorModeValue("gray.100", "generateKeyPair.700");
```

Comment:
what color is this "generateKeyPair.700?

Reply:
It is something that came with @magnusRBrok's changes in PR #22